src/openrct2/ride/RideData.cpp

```diff
@@ -312,6 +312,11 @@ constexpr const RideTypeDescriptor RideTypeDescriptors[RIDE_TYPE_COUNT] = {
     /* RIDE_TYPE_HYBRID_COASTER                     */ HybridCoasterRTD,
 };
 
+constexpr const RideComponentName& GetRideComponentName(const RideComponentType type)
+{
+    return RideComponentNames[EnumValue(type)];
```

Comment:
constexpr functions go in the header file.

Reply:
Is noted. thanks.